pkg/agent/agent_windows.go

```diff
@@ -369,21 +370,16 @@ func (i *Initializer) getTunnelPortLocalIP() net.IP {
 // The routes will be restored on OVS bridge interface after the IP configuration
 // is moved to the OVS bridge.
 func (i *Initializer) saveHostRoutes() error {
-	routes, err := util.GetNetRoutesAll()
+	family := antreasyscall.AF_INET
```

Comment:
add a comment here that mentions that we do not support IPv6 on Windows. You can even link to https://github.com/antrea-io/antrea/issues/5162.

Reply:
Done.